src/components/calendars/index.js

```diff
@@ -58,11 +59,13 @@ var CANONICAL_TICK = {
 };
 
 // Start on a Sunday - for week ticks
-// Discworld and Mayan calendars don't have 7-day weeks anyway so don't change them.
+// Discworld and Mayan calendars don't have 7-day weeks but we're going to give them
+// 7-day week ticks so start on our Sundays.
 // If anyone really cares we can customize the auto tick spacings for these calendars.
 var CANONICAL_SUNDAY = {
+    chinese: '2000-01-02',
     coptic: '2000-01-03',
-    discworld: '2000-01-01',
+    discworld: '2000-01-03',
```

Comment:
see the comment change above - I decided it would be better to keep our Sundays unless and until we support the native weeks in these calendars.